.buildkite/pipelines/on_merge.yml

```diff
@@ -532,6 +532,20 @@ steps:
         - exit_status: '-1'
           limit: 3
 
+  - command: .buildkite/scripts/steps/checks/capture_oas_snapshot.sh
+    label: 'Check OAS Snapshot'
+    agents:
+      image: family/kibana-ubuntu-2004
+      imageProject: elastic-images-prod
+      provider: gcp
+      machineType: n2-standard-2
+      preemptible: true
+    timeout_in_minutes: 60
+    retry:
+      automatic:
+        - exit_status: '-1'
```

Comment:
This is only going to check for agent lost signals, i.e. preemption.  Not sure if the goal was to catch other errors.

Reply:
I've added retries at the bash command call level, so we don't need to retry with Buildkite at this level, this check can remain for preemptions like for the other commands